src/components/site/EmphasizedImage.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const EmphasizedImage = props => (
+  <div className="dg_card-image" style={props.backgroundimage}>
+    <div className="dg_card-content">
+      <h4>{props.imageheader}</h4>
+      <h1>
+        <a href="#">{props.imagelink}</a>
+      </h1>
+    </div>
+  </div>
+);
+
+EmphasizedImage.propTypes = {
+  /** The image to be displayed */
+  backgroundimage: PropTypes.string.isRequired,
```

Comment:
`backgroundImageSrc` since this will update the image `src` property.

Reply:
done